### PR TITLE
Added Hohmann Transfer...

### PR DESCRIPTION
**Pull request by romeosarkar10x**
Self Explanatory 😂.

### CODE REVIEW

File: orbital_gui/__init__.py
Comment (Ayush1325) [inline orbital_gui/__init__.py:1]: Remove this file. It is unnecessary

File: orbital_gui/tasks/__init__.py
Comment (Ayush1325) [inline orbital_gui/tasks/__init__.py:1]: Again. Remove

File: orbital_gui/tasks/hohmann_transfer.py
Comment (Ayush1325) [inline orbital_gui/tasks/hohmann_transfer.py:1]: Should go inside `calculator_gui/tasks/*`. Not here

File: tests/__init__.py
Comment (Ayush1325) [inline tests/__init__.py:1]: Remove

File: tests/hohmann_transfer.py
Comment (Ayush1325) [inline tests/hohmann_transfer.py:1]: This is not a test. If you do not know how to write Django test, then do not add test.

**Review by Ayush1325 — CHANGES_REQUESTED** [review]
Fix Pylint errors. You can also run pylint locally for faster development:
```
pylint $(git ls-files '*.py')
```

File: orbital_gui/tasks/hohmann_transfer.py
Comment (romeosarkar10x) [reply orbital_gui/tasks/hohmann_transfer.py:1]: Image

File: tests/hohmann_transfer.py
Comment (romeosarkar10x) [reply tests/hohmann_transfer.py:1]: Is the "test" directory Correct?? If not where should I put my test file? Please clarify

File: orbital_gui/tasks/hohmann_transfer.py
Comment (Ayush1325) [reply orbital_gui/tasks/hohmann_transfer.py:1]: Read the README.md. Not whatsapp messages. Especially not ones not from me.

File: tests/hohmann_transfer.py
Comment (Ayush1325) [reply tests/hohmann_transfer.py:1]: Well, this file does not use [Django testing framework](https://docs.djangoproject.com/en/5.0/topics/testing/overview/) so it is not a test. As I said, if you do not know how to write real tests, just do not add them. It is not mandatory, just a nice to have. I am not going to merge something wrong though.

File: calculator_gui/tasks/hohmann_transfer.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    va = math.sqrt(mu * (2/r2 - 1/a))
+
+    # Delta-Vs required for the transfer
+    delta_v1 = vp - v1
+    delta_v2 = v2 - va
+
+    # Period of the transfer orbit
+    time_period = 2 * math.pi * math.sqrt(a**3 / mu)
+
+    return {
```
Comment (Ayush1325) [inline calculator_gui/tasks/hohmann_transfer.py]: Can you make the return data a [data class](https://realpython.com/python-data-classes/#basic-data-classes)?

**Review by Ayush1325 — CHANGES_REQUESTED** [review]
Everything else is good

File: calculator_gui/tasks/hohmann_transfer.py (hunk 1 above)
Comment (romeosarkar10x) [reply calculator_gui/tasks/hohmann_transfer.py]: Sure, just give me a few minutes.

**Pull request merged**